CMakeLists.txt

```diff
@@ -23,6 +24,14 @@
 #   cmake -D CMAKE_BUILD_TYPE=Debug ../SLiM
 #   make
 #
+
+# To build the Qt GUI, make sure you have the Qt5 Widgets, Core, and Gui libraries installed, then 
+# configure with:
+#   cd build
+#   cmake -D BUILD_QTSLIM=ON ../SLiM
+#   make -j10
```

Comment:
ditto

Reply:
Ah, I clicked "resolve conversation" on the other -j10 thread above without knowing exactly what it would do, and it appears to have deleted the whole thread.  Well, anyway: upon reflection I realized that if a user has even two cores -j10 is probably a win – maybe it's not the optimal number of threads, but at least you're using both cores.  And probably everybody has at least two cores nowadays.  So -j10 is fine with me, forget I raised the issue.  :->

Reply:
I will add clarification in the comments -- this is totally machine-dependent.